### PR TITLE
Document converting an array to a sequence

### PR DESCRIPTION
**Pull request by amaurremi**
Adds a paragraph to the [Sequences and arrays](https://dafny-lang.github.io/dafny/DafnyRef/DafnyRef#1051-sequences-and-arrays) section of the reference manual, showing how to convert an array to a sequence using the `[..]` operator.

<small>By submitting this pull request, I confirm that my contribution is made under the terms of the [MIT license](https://github.com/dafny-lang/dafny/blob/master/LICENSE.txt).</small>

### CODE REVIEW

File: docs/DafnyRef/Types.md [hunk 1]
```diff
@@ -1088,6 +1088,13 @@ with arrays where parallel assignment is needed:
   }
 ```
 
+To convert an array `x` to a sequence, use the `[..]` operator:
+```dafny
+  var intArr: array<int> := new int[] [1, 2, 3];
+  var intSeq: seq<int> := intArr[..];
+  assert intSeq == [1, 2, 3];
+```
```
Comment (cpitclaudel) [inline docs/DafnyRef/Types.md:1096]: Very nice!  But, should it be in the "Iterating over collections" chapter?

Comment (MikaelMayer) [reply docs/DafnyRef/Types.md:1096]: Can you either move it there https://dafny-lang.github.io/dafny/DafnyRef/DafnyRef#sec-array-types
or at least make a reference there that points to this?

Comment (amaurremi) [reply docs/DafnyRef/Types.md]: Yes, of course, moved the explanation and added a reference to it.

**Review by MikaelMayer — APPROVED** [review]
Great job ! I remember reading this doc once and not finding this conversion topic which would have been useful then.

**Pull request merged**